upload_cleanup.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/phrase/phraseapp-go/phraseapp"
+)
+
+type UploadCleanupCommand struct {
+	*phraseapp.Config
+	ID             string `cli:"arg required"`
+	SuppressPrompt bool   `cli:"opt --yes desc='Don’t ask for confirmation'"`
```

Comment:
I do not like `--yes` that much in this context but either go with `--confirm`, in that case we could just rename `SupressPrompt` to `Confirm` (as prompt is just the implementation of how to ask for confirmation anyway).